src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/ValidationRulePart.cshtml

```diff
@@ -0,0 +1,64 @@
+@using OrchardCore.Forms.Models
+@model ShapeViewModel<ValidationRulePart>
+@{
+    var type = Model.Value.Type;
+    var option = Html.Raw(Model.Value.Option);
+
+    var formElementPart = Model.Value.ContentItem.As<FormElementPart>();
+    var elementId = formElementPart.Id;
+}
+
+<script asp-src="~/OrchardCore.Forms/Scripts/validator.min.js" debug-src="~/OrchardCore.Forms/Scripts/validator.js" asp-name="validator" depends-on="jQuery" at="Foot"></script>
+
+<script at="Foot">
+    if (type != "none") {
+
+        var type = '@type';
+        var elementId = '#' + '@elementId';
+        var option = '@option';
+        var validateElementId = elementId + type;
+
+        if (window.validatorObject) {
+            var obj = {};
+            obj.type = type;
+            obj.elementId = '@elementId';
+            obj.option = option;
+            objs.push(obj);
+            window.validatorObject.push(obj);
+        } else {
+            window.validatorObject = new Array();
+            var obj = {};
+            obj.type = type;
+            obj.elementId = '@elementId';
+            obj.option = option;
+            window.validatorObject.push(obj);
+        }
+
+
+        $(elementId).bind("keyup", function (e) {
+            var re = false;
+            if (option == '') {
+               re = window.validator['@type']($(elementId).val());
+            } else {
+               re = window.validator['@type']($(elementId).val(), JSON.parse(option));
+            }
+
+            var docEle = document.getElementById(validateElementId);
+            if (re) {
+                if (docEle) {
+                    $(elementId).parent().parent().parent().find(":submit").removeAttr('disabled');
```

Comment:
Maybe find the submit button recursively instead of hard-coding a depth? Below too.